c/driver/framework/utility.cc

```diff
@@ -163,7 +163,6 @@ Status MakeGetInfoStream(const std::vector<InfoValue>& infos, ArrowArrayStream*
           } else {
             static_assert(!sizeof(T), "info value type not implemented");
           }
-          return status::Ok();
```

Comment:
doesn't this mean we don't have any return in this scenario?

Reply:
`else static assert` means there's no other possible path; it would fail to compile in the first place

Reply:
Ah, I didn't realize that the static_assert is pretty much guaranteed to fail. That makes sense

Reply:
static_assert is a compile time assertion